library_generation/owlbot/src/fix_poms.py

```diff
@@ -104,6 +107,43 @@ def update_cloud_pom(
         if m.find("{http://maven.apache.org/POM/4.0.0}artifactId") is not None
     ]
 
+    # as of July 2024, we have two dependencies that should be declared as
+    # test-scoped: grpc-google-common-protos and grpc-google-iam-v1. Only in
+    # java-storage, java-spanner and java-dataproc we keep them as they are
+    TEST_SCOPED_DEPENDENCIES = ["grpc-google-common-protos", "grpc-google-iam-v1"]
```

Comment:
I'm not sure I understand why we need this part. Is the template change not enough?

Reply:
The template is rendered only on new libraries, otherwise the function `update_cloud_pom()` is called.

https://github.com/googleapis/sdk-platform-java/blob/fdd9d27cdd8c42ddada5b41df7bd06e2e474ccf0/library_generation/owlbot/src/fix_poms.py#L532-L541